Makefile

```diff
@@ -468,6 +468,7 @@ endif
 		--no-add-mod-to-lib \
 		--no-auto-upgrade-dependency \
 		--no-web \
+		--no-rust-format \
```

Comment:
frb stopped formatting rust on CI for some reason, but everything is ok locally. This causes CI to fail on the `git status` check. I can try to figure out whats wrong, but in the end we dont really care whether rust-side frg generated glue is formatted.